api/router.go

```diff
@@ -58,7 +60,29 @@ func tokenCheckHandler() func(c *fiber.Ctx) error {
 	}
 }
 
-// @Description  Route for login
+// @Description  Route for refreshing the access token
+// @Tags         general
+// @Produce      json
+// @Success      200  {string} string
+// @Failure      401
+// @Router       /api/refresh [post]
+func refreshHandler() func(c *fiber.Ctx) error {
+	return func(c *fiber.Ctx) error {
+		token := c.Get("Authorization")
+		token = strings.Replace(token, "Bearer ", "", 1)
+
+		newAccess, err := auth.RefreshAccessToken(token)
+		if err != nil {
+			c.JSON(bson.M{"Message": err.Error()})
```

Comment:
Is there an upside of doing it this way vs. fiber.NewError() ?

Reply:
Nope just forgot we had refactored that. Will do that later for this and the other PR